src/graphql/account.rs

```diff
@@ -287,78 +287,58 @@ impl AccountMutation {
         Ok(username)
     }
 
-    /// Authenticates with the given username and password
+    /// Authenticate with the given username and password.
+    /// If the `last_signin_time` value of the `account` is `None`, the operation will fail, and
+    /// it should be guided to call `signInWithNewPassword` GraphQL API.
+    ///
+    /// # Guide
+    /// If the `signIn` GraphQL API response is the error message "a password change is
+    /// required to proceed", you can call the `signInWithNewPassword` GraphQL API.
+    ///
+    /// ```ignore
+    /// r#"mutation {
+    ///     signInWithNewPassword(username: "user", password: "pw", newPassword: "pw2") {
+    ///         token
+    ///     }
+    /// }"#
+    /// ```
+    ///
+    /// # Returns
```

Comment:
We usually don't mention returning values in a separate section. You can refer to the standard library to check how to write the Rust documentation.

Reply:
I'll delete it.

Reply:
If you deleted that part, that’s not what I meant. I just wanted to say that the return values should be mentioned according to the Rust documentation convention, like this:

```rust
///
/// Returns blah blah blah
///
```

- The title, `# Returns`, is not necessary.
- The sentence starts with `Returns`, a third-person singular verb.

Reply:
I'll be careful.